get-started/learn-more.md

```diff
@@ -3,28 +3,26 @@ layout: page
 title: "Get Started: Learn More"
 permalink: /get-started/learn-more/
 ---
-* Learn more about the Flutter framework
 
-{% comment %}
+Learn more about the Flutter framework:
+
+{%- comment %}
 Once the "Flutter for React" doc (etc) are done, add here.
```

Comment:
Oooh, this is ready! Could you add links for "Flutter for React Native", "Flutter for iOS", "Flutter for Xamarin.Forms" here?

Reply:
Adding them. What about "Flutter for Android Developers"?

Reply:
Done, including "Flutter for Android", sorted alphabetically.